akka-http2-support/src/test/scala/akka/http/impl/engine/http2/Http2ServerSpec.scala

```diff
@@ -678,6 +685,46 @@ class Http2ServerSpec extends AkkaSpec("" + "akka.loglevel = debug")
       }
     }
 
+    "expose synthetic headers" should {
+      "expose Remote-Address" in new TestSetup with RequestResponseProbes {
+
+        lazy val theAddress = "127.0.0.1"
+        lazy val thePort = 1337
+        override def modifyServer(server: BidiFlow[HttpResponse, ByteString, ByteString, HttpRequest, NotUsed]) =
+          BidiFlow.fromGraph(StreamUtils.fuseAggressive(server).withAttributes(
+            HttpAttributes.remoteAddress(Some(new InetSocketAddress(theAddress, thePort)))
+          ))
+
+        val probe = TestProbe()
+        override def handlerFlow: Flow[HttpRequest, HttpResponse, NotUsed] =
```

Comment:
If you use `RequestResponseProbes` you already have probes to use. Or is this just dead code?